Userland/Libraries/LibCrypto/Hash/SHA2.cpp

```diff
@@ -5,9 +5,17 @@
  * SPDX-License-Identifier: BSD-2-Clause
  */
 
+#include <AK/Platform.h>
 #include <AK/Types.h>
 #include <LibCrypto/Hash/SHA2.h>
 
+#if (ARCH(I386) || ARCH(X86_64)) && not defined(KERNEL)
```

Comment:
Why are you using `not defined` instead of `!defined` in this file?

Reply:
no particular reason, should I change that?

Reply:
I was just asking because I don't think we use alternative operator representations anywhere else?

Reply:
I'm also for changing it to `!`.